src/App.jsx

```
@@ -3,24 +3,24 @@ import AtroposCard from './components/atropos'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
+	const [count, setCount] = useState(0)
```

ESLint

[error] 'count' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)